lib/generators/thredded/install/templates/initializer.rb

```diff
@@ -99,3 +99,14 @@
 #     end
 #   end
 # end
+
+# ==> View hooks
+#
+# Customize the UI before/after/replacing individual components.
+# See the full list of view hooks by running:
+#
+#     $ grep view_hooks "$(bundle show thredded)"/**/*.html.erb.
```

Comment:
nit: only works on zsh-like shells (** doesn't work on bash for example).
Alternative (or just state "on zsh:"):

```grep view_hooks -R --include *.html.erb "$(bundle show thredded)"```